projects/common/vpk180/vpk180_system_bd.tcl

```diff
@@ -0,0 +1,221 @@
+ad_connect  sys_cpu_clk sys_rstgen/slowest_sync_clk
+ad_connect  sys_cips/pl0_resetn sys_350m_rstgen/ext_reset_in
+ad_connect  sys_350m_clk sys_350m_rstgen/slowest_sync_clk
+
+ad_connect  sys_cpu_reset sys_rstgen/peripheral_reset
+ad_connect  sys_cpu_resetn sys_rstgen/peripheral_aresetn
+ad_connect  sys_350m_reset sys_350m_rstgen/peripheral_reset
+ad_connect  sys_350m_resetn sys_350m_rstgen/peripheral_aresetn
+
+# gpio
+ad_connect gpio0_i sys_cips/lpd_gpio_i
+ad_connect gpio0_o sys_cips/lpd_gpio_o
+ad_connect gpio0_t sys_cips/lpd_gpio_t
+
+# gpio extension witn 64 IOs
+ad_ip_instance axi_gpio axi_gpio
+ad_ip_parameter axi_gpio CONFIG.C_IS_DUAL 1
+ad_ip_parameter axi_gpio CONFIG.C_GPIO_WIDTH 32
+ad_ip_parameter axi_gpio CONFIG.C_GPIO2_WIDTH 32
+ad_ip_parameter axi_gpio CONFIG.C_INTERRUPT_PRESENT 1
+
+ad_connect gpio1_i axi_gpio/gpio_io_i
+ad_connect gpio1_o axi_gpio/gpio_io_o
+ad_connect gpio1_t axi_gpio/gpio_io_t
+ad_connect gpio2_i axi_gpio/gpio2_io_i
+ad_connect gpio2_o axi_gpio/gpio2_io_o
+ad_connect gpio2_t axi_gpio/gpio2_io_t
+
+ad_cpu_interconnect 0x44000000 axi_gpio
+
+ad_cpu_interrupt ps-0 mb-xx axi_gpio/ip2intc_irpt
+
+## generic system clocks&resets pointers
+
+set sys_cpu_clk           [get_bd_nets sys_cpu_clk]
+set sys_dma_clk           [get_bd_nets sys_350m_clk]
+
+set sys_cpu_reset         [get_bd_nets sys_cpu_reset]
+set sys_cpu_resetn        [get_bd_nets sys_cpu_resetn]
+set sys_dma_reset         [get_bd_nets sys_350m_reset]
+set sys_dma_resetn        [get_bd_nets sys_350m_resetn]
+
+# spi
+
+ad_ip_instance xlconcat spi0_csn_sources
+ad_ip_parameter spi0_csn_sources config.num_ports {3}
+ad_connect spi0_csn_sources/dout spi0_csn
+
+ad_connect  sys_cips/spi0_sck_o spi0_sclk
+ad_connect  sys_cips/spi0_sck_i GND
+ad_connect  sys_cips/spi0_io0_o spi0_mosi
+ad_connect  sys_cips/spi0_io0_i spi0_miso
+ad_connect  sys_cips/spi0_io1_i GND
+ad_connect  sys_cips/spi0_ss_o  spi0_csn_sources/in0
+ad_connect  sys_cips/spi0_ss1_o spi0_csn_sources/in1
+ad_connect  sys_cips/spi0_ss2_o spi0_csn_sources/in2
+ad_connect  sys_cips/spi0_ss_i  VCC
+
+ad_ip_instance xlconcat spi1_csn_sources
+ad_ip_parameter spi1_csn_sources config.num_ports {3}
+ad_connect spi1_csn_sources/dout spi1_csn
+
+ad_connect  sys_cips/spi1_sck_o spi1_sclk
+ad_connect  sys_cips/spi1_sck_i GND
+ad_connect  sys_cips/spi1_io0_o spi1_mosi
+ad_connect  sys_cips/spi1_io0_i spi1_miso
+ad_connect  sys_cips/spi1_io1_i GND
+ad_connect  sys_cips/spi1_ss_o  spi1_csn_sources/in0
+ad_connect  sys_cips/spi1_ss1_o spi1_csn_sources/in1
+ad_connect  sys_cips/spi1_ss2_o spi1_csn_sources/in2
+ad_connect  sys_cips/spi1_ss_i  VCC
+
+# system id
+
+ad_ip_instance axi_sysid axi_sysid_0
+ad_ip_instance sysid_rom rom_sys_0
+
+ad_connect  axi_sysid_0/rom_addr   	    rom_sys_0/rom_addr
+ad_connect  axi_sysid_0/sys_rom_data   	rom_sys_0/rom_data
+ad_connect  sys_cpu_clk                 rom_sys_0/clk
+
+ad_cpu_interconnect 0x45000000 axi_sysid_0
+
+# interrupts
```

Comment:
I think the interrupt from line 168 should go underneath this comment, right?

If not, then remove the interrupts comment and the line before that

Reply:
The interconnects from 166 and 221 should be placed one after the other, and 168 should be underneath this comment